cardano_node_tests/utils/dbsync_utils.py

```diff
@@ -985,3 +985,25 @@ def check_drep_deregistration(
     ), f"Wrong deposit value for deregistered DRep {drep.drep_id} in db-sync"
 
     return drep_data
+
+
+def check_votes(votes: governance_utils.VotedVotes, txhash: str) -> None:
+    """Check votes in db-sync."""
```

Comment:
As this will be used in tests directly, we need to check we have db-sync available.